backend/src/main/migrations/20220113235621-edit-task-projectId-required.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.changeColumn('Task', 'projectId', {
+      allowNull: true,
+      type: Sequelize.BIGINT,
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+      references: {
+        model: 'project',
+        key: 'id'
+      }
+    });
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.changeColumn('Task', 'projectId', {
```

Comment:
why are you doing a "change column" when doing the "down"?

Reply:
Same thing as the other down